.travis.yml

```diff
@@ -1,9 +1,17 @@
 language: go
 go:
   - 1.4
+  - 1.5.3
+  - 1.6
+  - tip
+
+matrix:
+  allow_failures:
+    - go: tip
 
 install:
  - go get golang.org/x/tools/cmd/cover
 
 script:
  - ./build-check
```

Comment:
AFAICT build-check effectively just runs `gofmt`, so IMHO we can get rid of it all-together.